canonicals.js

```diff
@@ -0,0 +1,53 @@
+var APP = APP || {};
```

Comment:
I considered requirejs or browserify, but I think the requirejs syntax is extremely bloated and I don't like it. Browserify is much nicer IMO but would need a build-step so I figured I'd just go bare-bone.

Maybe just scrap the modules entirely and put everything in a single file is just as good? Would save 2 requests as well.